release/long_running_tests/app_config.yaml

```diff
@@ -0,0 +1,15 @@
+base_image: "anyscale/ray-ml:1.4.0"
```

Comment:
Isn't this supposed to be for nightly?

Reply:
This is just the base image - we will install the correct ray wheels in the post build cmds